website/docs/r/compute_router_interface.html.markdown

```diff
@@ -8,8 +8,10 @@ description: |-
 
 # google\_compute\_router_interface
 
-Manages a Cloud Router interface. For more info, read the
-[documentation](https://cloud.google.com/compute/docs/cloudrouter).
+Manages a Cloud Router interface. For more information see
+[the official documentation](https://cloud.google.com/compute/docs/cloudrouter)
```

Comment:
Not needed for LGTM but nice-to-have: is there a header we can link to in the docs that is more relevant than the Cloud Router page as a whole?